### DISCUSSION TITLE
Access Quinoa generated resources manually

### DISCUSSION THREAD

**Discussion by rasmushaglund** [quarkiverse/quarkus-quinoa#424] [General]
Hello, I'm looking into a use case where I have a web created with Qute/Renarde and want to use Quinoa built components in some of those pages.

As far as I know Quinoa requires a start index.html page that includes all required javascripts, and the generated js-resource contains the hash in the file name, making it hard to access manually.

Is there any obvious ways to include these Quinoa generated resources from, for example, a Qute template? 

Thank you, and thanks for a great extension!

**Comment by ia3andy** [comment]
Hey Rasmus,

I have the perfect solution for you: 
https://github.com/quarkiverse/quarkus-web-bundler/

There is exactly what you are looking for regarding components.

Here is an example usage: https://github.com/ia3andy/renotes/tree/main/src/main/resources/web/components

I will soon start working on the guide for it.

Let me know :)

**Reply by rasmushaglund** [reply]
That sounds super, I'll check it out!
